core-web/apps/dotcms-ui/src/app/view/components/_common/dot-alert-confirm/dot-alert-confirm.ts

```diff
@@ -20,7 +22,8 @@ export class DotAlertConfirmComponent implements OnInit, OnDestroy {
             .pipe(takeUntil(this.destroy$))
             .subscribe(() => {
                 setTimeout(() => {
-                    this.confirmBtn.nativeElement.focus();
+                    const btn = this.confirmBtn || this.acceptBtn;
+                    btn.nativeElement.focus();
```

Comment:
More likely: https://angular.io/api/core/AfterViewInit

Reply:
Or maybe https://primefaces.org/primeng/autofocus?

Reply:
this alert dialog is loaded in the app in general, and the show/hide depends on a boolean Subject that enables it or not, so afterviewInit can not be used, neither the `autofocus`